h2o-core/src/main/java/hex/grid/FilteredHyperSpaceIterator.java

```diff
@@ -0,0 +1,80 @@
+package hex.grid;
+
+import hex.Model;
+import hex.grid.filter.PermutationFilter;
+
+public class FilteredHyperSpaceIterator<MP extends Model.Parameters> implements HyperSpaceWalker.HyperSpaceIterator<MP> {
```

Comment:
_hyperSpaceIterator, _permutationFilter and _maxHyperSpaceSize can be final

Reply:
@honzasterba _maxHyperSpaceSize is final. will add for two others

Reply:
@honzasterba fixed